marathon/fetcher.go

```diff
@@ -26,7 +26,19 @@ func NewAppFetcher(u string) (*AppFetcher, error) {
 	}, nil
 }
 
-// FetchApps fetches apps with specific label set to specific value
+// Apps fetches all apps from marathon.
+func (a *AppFetcher) Apps() ([]marathon.Application, error) {
```

Comment:
Changed this but was worried about other things that depend on this.  I guess there isn't really anything that would use zoidberg as a library though.